ansible/roles/backups/tasks/main.yml

```diff
@@ -5,7 +5,7 @@
     dest: "/etc/cron.d/create_blobdb_backup.sh"
     group: root
     owner: root
-    mode: 700
+    mode: 0700
```

Comment:
ansible file modes need to have a leading zero, so i went back and updated existing files that lacked them.